test/community/label_propagation.jl

```diff
@@ -0,0 +1,12 @@
+n=10
+g10 = CompleteGraph(n)
+z = copy(g10)
+for k=2:5
```

Comment:
I think this loop is making a path of clique graphs where the cliquesize is 10 and the number of cliques ranges from 2:6, and testing each one to assign each clique the same value. Is that right?